modules/sdk/project-templates/project-templates-npm-billboardjs-portlet/src/main/resources/archetype-resources/package.json

```diff
@@ -3,13 +3,11 @@
 		"billboard.js": "1.1.1"
 	},
 	"devDependencies": {
-		"babel-cli": "6.26.0",
-		"babel-preset-es2015": "6.24.1",
-		"babel-preset-liferay-project": "1.6.1",
-		"liferay-npm-bundler": "1.6.1",
-		"liferay-npm-bundler-preset-standard": "1.6.1"
+		"babel-cli": "^6.26.0",
+		"babel-preset-env": "^1.7.0",
+		"liferay-npm-bundler": "^2.0.0"
 	},
-	"main": "js/index.js",
+	"main": "lib/index.es.js",
 	"name": "${artifactId}",
 	"scripts": {
 		"build": "babel --source-maps -d#if (${buildType} != "gradle") target/classes/META-INF/resources#else build/resources/main/META-INF/resources#end src/main/resources/META-INF/resources && liferay-npm-bundler"
```

Comment:
So in the angular portlet we no longer need to run babel.  But we still need to invoke it here?

Reply:
Yes. Angular is the only one that seems to have babel removed. All the other ones still invoke babel and depend on it